index.html

```diff
@@ -4,8 +4,11 @@
     <meta charset="UTF-8">
     <meta name="description" content="Trouvez votre hébergement pour des vacances de rêve">
     <meta name="keywords" content="Hébergements, Activités, Tourisme, nature, ville, vacances, voyage">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/js/solid.min.js" integrity="sha512-JkeOaPqiSsfvmKzJUsqu7j2fv0KSB6Yqb1HHF0r9FNzIsfGv+zYi4h4jQKOogf10qLF3PMZEIYhziCEaw039tQ==" crossorigin="anonymous"></script>
```

Comment:
Why do you have two fontawesome ?

Reply:
j'avais oublié que j'avais importé mon kit pour voir les icons pendant que je faisais le HTML.